gulp/ts.js

```diff
@@ -7,7 +7,7 @@ var ts = require('gulp-typescript');
 var tslint = require('gulp-tslint');
 
 gulp.task('ts-build', function () {
-    return gulp.src(['at-*.ts', 'ts/**/*.d.ts', 'test/**/*.ts']).pipe(ts({
+    return gulp.src(['at-*.ts', '*.d.ts', 'typings/**/*.d.ts', 'test/**/*.ts']).pipe(ts({
```

Comment:
I think that we should grab only one file from typings - the one that acts as ToC… But I know typings is changing very fast and I may be wrong.

You can try with a `a wildcard to grab all definition files at the root` but I think that it is not future-proof way.